release/golden_notebook_tests/workloads/torch_tune_serve_test.py

```diff
@@ -247,19 +238,19 @@ def stop_ray():
     if addr is not None and addr.startswith("anyscale://"):
         client = ray.init(address=addr, job_name=job_name)
     else:
-        client = ray.init(address="auto")
+        client = ray.init()
```

Comment:
address="auto" doesn't work? it should be the same no?

Reply:
Ah, my bad, I removed this for testing. Thanks for catching it!